cmd/fabric-ca-server/main_test.go

```diff
@@ -118,20 +107,48 @@ func TestErrors(t *testing.T) {
 		{[]string{cmdName, "start", "-c", startYaml, "-b", "user:pass", "ca.key"}, "Unrecognized arguments found"},
 	}
 
-	for _, e := range errorCases {
-		errorTest(&e, t)
-		_ = os.Remove(initYaml)
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.expected, func(t *testing.T) {
+			err := RunMain(tt.cmd)
+			assert.Error(t, err, tt.expected)
+			os.Remove(initYaml)
+		})
 	}
 }
 
 func TestOneTimePass(t *testing.T) {
-	testDir := "oneTimePass"
-	os.RemoveAll(testDir)
-	defer os.RemoveAll(testDir)
-	// Test with "-b" option
-	err := RunMain([]string{cmdName, "init", "-b", "admin:adminpw", "--registry.maxenrollments", "1", "-H", testDir})
-	if err != nil {
-		t.Fatalf("Failed to init server with one time passwords: %s", err)
+	t.Parallel()
+
+	tests := []struct {
+		testName string
+		cmd      []string
+	}{
+		{
+			testName: "When identity has user and pass",
+			cmd:      []string{cmdName, "init", "-b", "admin:adminpw", "--registry.maxenrollments", "1", "-H", os.TempDir()},
+		},
+		{
+			testName: "When identity has user and passfile",
+			cmd:      []string{cmdName, "init", "-f", "pwFile", "--registry.maxenrollments", "1", "-H", os.TempDir()},
```

Comment:
Where does the user come for this test command? I'm not super familiar with fabric-ca but I was expecting to see the -b option here for the username.

Reply:
ah, this was oversight in the test definition. The test passes because there are no checks around the user existence so the value for user remains `<<<ADMIN>>>`.